src/database/queries/addData.js

```diff
@@ -0,0 +1,22 @@
+const connection = require('../config/db_connection');
+
+const addUser = (userInfo) => {
+  const queryObj = {
+    text: 'insert into users (firstname, lastname, mobile_number, email, freelancer_url, photo_url,birthday, specalization_id,city,password ) values ($1, $2, $3, $4, $5, $6, $7, $8, $9, $10) RETURNING *',
+    values: [
+      userInfo.firstname,
```

Comment:
I think this query wrong because we don't need to insert city and birthday?
and we give you two ways to follow it #62  !!!!
then if you change it you should change test else